src/llms/ollama_client.py

```diff
@@ -0,0 +1,26 @@
+import ollama
+
+from .base_client import BaseClient
+
+
+class OllamaClient(BaseClient):
+    def __init__(self):
+        super().__init__()
+
+    def chat_completion(
+        self, model: str, system: str, user: str, parameters: dict
```

Comment:
Could we initial the client with desired model? 
Or do we want to have the ability to change the model for each `chat_completion`

Reply:
moved to `init`